UnityProject/Assets/Scripts/Items/ThrowableBreakable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AddressableReferences;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Items
+{
+	public class ThrowableBreakable : MonoBehaviour
+	{
+		[SerializeField]
+		private GameObject brokenItem = default;
+
+		[SerializeField, Range(0, 100)]
+		private int chanceToBreak = 100;
+
+		[SerializeField]
+		private AddressableAudioSource soundToPlay = SingletonSOSounds.Instance.GlassBreak01;
```

Comment:
can you check quickly that if you change it on the prefab it doesn't change it for the Bass sound

Reply:
sorry what do you mean by this?

Reply:
so, if you change the prefab, does it change the SO?

Reply:
it is changing the prefab. If it was changing a value on the SO it would be persistent.